vipps-recurring-api.md

```diff
@@ -709,6 +710,15 @@ the charge states returned by
 | 8 | `CANCELLED` | The charge has been cancelled. |
 | 9 | `PROCESSING` | The charge is currently being processed by Vipps. Normal processing takes less than 1 second, but in some cases they can stay in this status for several minutes |
 
+### Example charge flows
+Scenario: retryDays = 0, user does not have funds:
+`PENDING` -> `DUE` -> `PROCESSING` -> `DUE` -> `PROCESSING` -> `FAILED`
+
+Scenario: user does not have funds on first attempt, but second attempt is successful:
+`PENDING` -> `DUE` -> `PROCESSING` -> `DUE` -> `PROCESSING` -> `CHARGED`
+
+**Please note:** Since charges are polled, it is possible that the charge status can appear to perform strange transitions, e.g `PROCESSING` -> `CHARGED` or even `PROCESSING` -> `REFUNDED` depending on your systems. Usually `PROCESSING` is not seen as a charge usually only has this status for a brief period of time.
```

Comment:
```suggestion
**Please note:** Since charges are polled, it is possible that the charge status can appear to perform strange transitions, e.g `PROCESSING` -> `CHARGED` or even `PROCESSING` -> `REFUNDED` depending on your systems. Usually `PROCESSING` is not seen as a charge usually only has this status for a brief period of time, but merchants should always make sure they can handle this status.
```